lib/auth/in_memory/vault.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
I understand your point. My reason for moving the whole backend is that  we need the in_memory backend and vault.js for testing the whole thing.

I've set vault.js as the default authentication handler, so S3 can just import Arsenal and use it for signature generation. Looks transparent and easy to me.